plugins/woocommerce-blocks/assets/js/blocks/product-filters/edit.tsx

```diff
@@ -119,6 +123,65 @@ export const Edit = ( {
 		''
 	);
 
+	const [
+		productFiltersOverlayNavigationAttributes,
+		setProductFiltersOverlayNavigationAttributes,
+	] = useLocalStorageState< Record< string, unknown > >(
+		'product-filters-overlay-navigation-attributes',
+		{}
+	);
+
+	useEffect( () => {
+		const overlayClientIds = select( 'core/block-editor' ).getBlocksByName(
+			'woocommerce/product-filters-overlay-navigation'
+		);
+
+		const overlayBlockAttributes = select(
+			'core/block-editor'
+		).getBlockAttributes( overlayClientIds[ 0 ] );
```

Comment:
I believe it would be better to select only the blocks that are inner blocks of the current Product Filters block instance. And also, instead of selecting them by the position index, perhaps it would be better to filter them by the `attributes.triggerType === 'open-overlay` instead. This way, we can prevent the error that I mentioned in my testing results.